lib/rust/ensogl/component/text/src/font/family/src/lib.rs

```diff
@@ -190,4 +189,77 @@ impl NonVariableFaceHeader {
     pub fn new(width: Width, weight: Weight, style: Style) -> Self {
         Self { width, weight, style }
     }
+
+    /// Distance between two font variations. It is used to find a closest variations if the
+    /// provided is not available.
```

Comment:
I renamed it to `similarity_distance`. The problem iwth the word `metric` is that we would need to name the metric. It could be `distance_in_the_similarity_metric`, but IMO this is too wordy.